<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignOn</class>
 <widget class="QWidget" name="SignOn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 243, 225, 100);
border-width: 1px 1px 1px 1px; 
border-style: solid; 
border-color: rgba(255, 243, 225, 150);
border-radius:6px;</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="userLabel">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>user</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="pwLabel">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="pwLabel2">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>password</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="userLineEdit">
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;Consolas&quot;;</string>
           </property>
           <property name="text">
            <string>input username</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pwLineEdit1">
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;Consolas&quot;;</string>
           </property>
           <property name="text">
            <string>input password</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pwLineEidt2">
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;Consolas&quot;;</string>
           </property>
           <property name="text">
            <string>ensure password</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="signOnButton">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">alternate-background-color: rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string>Sign On</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="topButtonSignIn">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color : rgb(153,153,153);</string>
       </property>
       <property name="text">
        <string>Sign In</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="topButtonSignOn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color : rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Sign On</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="GifWidget" native="true">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>239</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
   <widget class="QLabel" name="charLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jenLabel">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>40</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bulLabel">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>51</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
